tendrl/commons/objects/__init__.py

```diff
@@ -162,6 +162,16 @@ def save(self, update=True, ttl=None):
                 NS._int.wreconnect()
                 NS._int.wclient.write(item['key'], item['value'], quorum=True)
 
+    def load_all(self):
+        value = '/'.join(self.value.split('/')[:-1])
+        etcd_resp = NS._int.client.read(value)
```

Comment:
You might need to handle EtcdConnectionFailed as in https://github.com/Tendrl/commons/pull/621/files/6568a8bd1357aef13a3411c4a43577fdc8a9985e#diff-fbc8ea2e973a12d3591504a5230f8186R186